### DISCUSSION TITLE
KtonDAO is online.

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1571] [General]
Step 1: Visit KtonStakingV1 and connect your wallet on the Darwinia chain:
https://staking-v1.ktondao.xyz/
Claim and then unstake your KTON.

Step 2: Visit KtonStakingV2 and connect your wallet on the Darwinia chain:
https://staking.ktondao.xyz/
Approve and stake. You will receive a non-transferable gKTON token, which can be used for voting on https://www.tally.xyz/gov/ktondao.

Step 3: Wait for a moment (Tally needs time to finish indexing before displaying your gKTON balance), then visit https://www.tally.xyz/gov/ktondao and connect your wallet on the Darwinia chain.
Delegate to yourself or someone else before they vote on your behalf.

**Comment by danairey** [comment]
why have the rewards not moved across, i moved first but barely earn any ring

**Comment by 2xJDubs** [comment]
Answered

**Comment by hackfisher** [comment]
> why have the rewards not moved across, i moved first but barely earn any ring

V2 pool just warmed up with 1 session reward, they will become the same reward as more session rewards are coming.

With v2, you can vote right now, rewards to v1 might be removed by some proposal months later.

**Comment by danairey** [comment]
Yes but how much longer are we talking till the rewards move over, i would rather stay in v1 pool but I cant. All rewards should move across now forcing people to move, the slow movers are the ones who should lose out.

**Reply by hackfisher** [reply]
Use the vercel link in this Pull request if you want to interact with & rejoin V1 contracts.

https://github.com/darwinia-network/kton-staking-ui/pull/19#issuecomment-2167006489

New rewards to v1 and v2 are exactly the same, V1 might still have some extra existing trailing rewards(sharing from previous sessions), but extra part will soon reduce to zero in several days.

**Comment by danairey** [comment]
Thanks
